snowball.go

```diff
@@ -61,7 +52,6 @@ type Snowball struct {
 
 func NewSnowball(opts ...SnowballOption) *Snowball {
 	s := &Snowball{
-		beta:       SnowballDefaultBeta,
```

Comment:
How often do we change the Beta for one Snowball instance but not every instance tied to the configuration ?  I see we do it in the test, and it requires some hackery to work around, but is that the only case where we do care ?

Maybe we can pass in a pointer to our ledger's `conf` object instead, to pull out that Beta ?

Reply:
not sure which hackery you're about)
snowballBeta should be changed once for all node's components. but since its used only in snowball instance, its relevant only for particular snowball entity, so it doesn't really matter. its used here in such way, because snowball is created once per node lifetime, so it can't be passed as parameter or something.
I'm not sure whats the usecase for changing those values in general, i guess its just to make testing network configurations easier. so i don't think its ever will be used in "production"
as for pointer - basically current implementation is a wrapper around pointer.